apps/condo/domains/billing/access/BillingIntegrationLog.js

```diff
@@ -11,20 +10,12 @@ async function canReadBillingIntegrationLogs ({ authentication: { item: user } }
     if (!user) return false
     if (user.isAdmin) return true
     return {
-        // TODO(pahaz): wait https://github.com/keystonejs/keystone/issues/4829 (no access check!)
-        // context: {
-        //     OR: [
-        //         {
-        //             organization: {
-        //                 employees_some: {
-        //                     user: { id: user.id },
-        //                     role: { canManageIntegrations: true },
-        //                 },
-        //             },
-        //         },
-        //         { integration: { accessRights_some: { user: { id: user.id } } } },
-        //     ],
-        // },
+        context: {
+            OR: [
+                { organization: { employees_some: { user: { id: user.id }, role: { canManageIntegrations: true }, deletedAt: null, isBlocked: false } } },
```

Comment:
role should not be deleted at

Reply:
I think we can leave it for now, as we are opt to recreate access and soft delete stuff anyway?